src/entities/Customer.java

```diff
@@ -188,10 +188,10 @@ private String makeBooking(ArrayList<Cinema> cinemaDB, ArrayList<Movie> movieDB)
 			Ticket newTicket = new Ticket(mov, cinema, this, showtime, cinema.getSeat(title+showtime.toString(), index), TID, this.username, this.email, this.mobileNumber);
 			this.bookings.add(newTicket);
 			System.out.println("The price of your ticket is: " + newTicket.getPrice());
-			System.out.println("Ticket booked successfully. A confirmation has been sent to the provided email and phone number");
 			break;
 		}
-	}		
+	}
+		return "Ticket booked successfully. A confirmation has been sent to the provided email and phone number";
```

Comment:
This method requires a return method. So I simply converted the last print statement into a return statement.